GOVERNANCE.md

```diff
@@ -0,0 +1,78 @@
+# Eclipse iceoryx governance
+
+Eclipse iceoryx is governed democratically by its maintainers under the umbrella of [OpenADx](https://openadx.eclipse.org/).
+This document describes the processes additionally to the [Eclipse process](https://www.eclipse.org/projects/handbook/).
+
+## Maintainers
+
+An up to date list of the maintainers can be found at the [Eclipse project page](https://projects.eclipse.org/projects/technology.iceoryx/who).
+
+## Bi-weekly meetup
+
+Bi-weekly every Thursday for 1 hour. Everyone is welcome to join.
+
+```
+08:00 PST (GMT - 8)
+16:00 GMT
+17:00 CET (GMT + 1)
+21:30 IST (GMT + 5.5)
+23:00 CST (GMT + 7)
+```
+
+### Topics
+
+* Propose features
+  * E.g. present and discuss a design document
+* Discuss pull requests without going into details
+  * E.g. a small demo
+* Steer direction of Eclipse iceoryx
+  * Go over the current release board under Projects
+  * E.g. go over the milestone list and move the issues between the three prio's
+  * In case of disagreement a vote is casted amongst the attendees
```

Comment:
The Eclipse process says that people who did a lot for the project can become committers after an election. The committers should have a stronger voice than just attendees

Reply:
Fully agree, only elected committers can propose new committers and elect them,

Reply:
Removed the line. I believe we always can find a solution by talking about it :-)